No zero-amount transfers

tbg opened the pull request:
They might still be valuable to test, but currently they mostly
distract during debugging.

Don't actually have Jepsen set up, so please check that this is actually Clojure (it's been a while).

nvb commented on cockroachdb/src/jepsen/cockroach.clj:
```diff
@@ -908,7 +908,7 @@
      :value {:from   (rand-int n)
              :to     (rand-int n)
-             :amount (rand-int 5)}}))
+             :amount (1 + rand-int 4)}}))
```
I've never written Clojure, but from the minuscule knowledge I have of Racket, I think the plus sign may come in front.

tbg replied:
Yeah, that rings a bell. Changed.

On Sat, Feb 27, 2016 at 1:21 AM Nathan VanBenschoten <
notifications@github.com> wrote:

> In cockroachdb/src/jepsen/cockroach.clj
> https://github.com/cockroachdb/jepsen/pull/9#discussion_r54329289:
> 
> > @@ -908,7 +908,7 @@
> >       :f     :transfer
> >       :value {:from   (rand-int n)
> >               :to     (rand-int n)
> > -             :amount (rand-int 5)}}))
> > -             :amount (1 + rand-int 4)}}))
> 
> I've never written Clojure, but from the minuscule knowledge I have of
> Racket, I think the plus sign may come in front.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/cockroachdb/jepsen/pull/9/files#r54329289.

tbg committed:
No zero-amount transfers

They might still be valuable to test, but currently they mostly
distract during debugging.
